src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -20,10 +20,17 @@
            [org.apache.commons.fileupload.util LimitedInputStream]))
 
 (def min-supported-commands
```

Comment:
looks like this could be derived from supported-command-versions above? (yes, I know it's preexisting)